bitmap-team-TeamName/test/bmp-file-helper-test.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const expect = require('chai').expect;
+const fs = require('fs');
+
+describe('bmp-file-helper Module', function() {
+  describe('Expecting readFile to output a buffer.', function() {
+    it('should be a pseudo-object ', function(done) {
+      fs.readFile(`${__dirname}/../img/palette-bitmap.bmp`, function(err, data) {
+        expect(err).to.equal(null);
+        expect(typeof data).to.equal('object');
+        done();
+      });
+    });
+  });
+});
```

Comment:
@npisciotti1 @radenska @sneakyneeks2 @darms this includes a decent test - as stated above, a better approach would be to encapsulate your `readFile` calls into a `read-file-helper` that can be exported and used on demand, including in your tests